.rubocop.yml

```diff
@@ -27,3 +27,7 @@ Layout/LineLength:
 Rails/Delegate:
   Exclude:
     - app/controllers/api/concerns/act_as_api_request.rb
+
+Rails/LexicallyScopedActionFilter:
+  Exclude:
+    - app/controllers/api/v1/api_controller.rb
```

Comment:
why not following the rules?

Reply:
This is to be able to do this https://github.com/rootstrap/rails_api_base/pull/253/files#diff-e3dabcd50af3f695337100894a51d413R7

which is what we are currently doing in ApplicationController but somehow rubocop is not complaining about

Reply:
what are the conflicting lines?

Reply:
These two https://github.com/rootstrap/rails_api_base/pull/253/files#diff-e3dabcd50af3f695337100894a51d413R7-R8